### PR TITLE
Input value PropType can be array

### LINKED ISSUE #429: Input value PropTypes should allow for array values
When the Input component handles multiple values, the PropType for value should allow for PropType.array.

### PR DESCRIPTION
**Pull request by AleF83**
fixes #429

### CODE REVIEW

File: test/InputSpec.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ describe('<Input />', () => {
     });
   });
 
+  context('#text with value', () => {
+    it('renders an input with value', () => {
```
Comment (alextrastero) [inline test/InputSpec.js]: thanks for this, could we add another assertion for the `array` type?

Comment (alextrastero) [reply test/InputSpec.js]: like what we expect to appear when you add 2 values in the `value` prop

Comment (AleF83) [reply test/InputSpec.js]: I saw that there are another tests that check 'select' element that already validate it so I didn't add test for this case.

Comment (alextrastero) [reply test/InputSpec.js]: I think we should test that at least it doesn't break

Comment (AleF83) [reply test/InputSpec.js]: I've added test case

**Comment by alextrastero** [comment]
Smashing! Thanks

**Comment by alextrastero** [comment]
Released on `1.1.1`

**Pull request merged**